### PR TITLE
This template allows deploying seafile server 6.1.1 on Azure Ubuntu VM

### PR DESCRIPTION
**Pull request by ostclilideng**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by ostclilideng** [comment]
@gatneil could you please help take a look at why all checks failed. What is the fail for? Thanks!

File: ubuntu-netdisk-setup/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+    "nicName": "myVMNic1",
+    "addressPrefix": "10.0.0.0/16",
+    "subnetName": "Subnet",
+    "subnetPrefix": "10.0.0.0/24",
+    "publicIPAddressName": "myPublicIP",
+    "publicIPAddressType": "Dynamic",
+    "vmName": "MyUbuntuVM",
+    "virtualNetworkName": "MyVNET",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "_artifactsLocation": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/ubuntu-netdisk-setup/",
```
Comment (gatneil) [inline ubuntu-netdisk-setup/azuredeploy.json]: Please see best practice checklist 2: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

File: ubuntu-netdisk-setup/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+          }
+        ],
+        "networkSecurityGroup": 
+        {
+            "id": "[resourceId('Microsoft.Network/networkSecurityGroups','mysg')]"
+        }
+      }
+    },
+    {
+    "apiVersion": "2015-05-01-preview",
```
Comment (gatneil) [inline ubuntu-netdisk-setup/azuredeploy.json]: is there a non-preview api version we can use?

**Comment by gatneil** [comment]
@ostclilideng I believe this is a bug in the CI; having this investigated; in the mean time, there are just a couple comments on the PR; once these are addressed, we can merge the changes assuming you've tested the template manually :)

File: ubuntu-netdisk-setup/azuredeploy.json (hunk 1 above)
Comment (ostclilideng) [inline ubuntu-netdisk-setup/azuredeploy.json]: Do you mean put  _artifactsLocation into parameter?

Comment (gatneil) [inline ubuntu-netdisk-setup/azuredeploy.json]: Yup :)

Comment (ostclilideng) [inline ubuntu-netdisk-setup/azuredeploy.json]: Thanks Neil, updated in new commit, and test pass

File: ubuntu-netdisk-setup/azuredeploy.json (hunk 2 above)
Comment (ostclilideng) [inline ubuntu-netdisk-setup/azuredeploy.json]: Thanks Neil, updated in new commit, and test passLeave a comment

**Comment by gatneil** [comment]
Awesome! The changes look good. Merging :)

**Pull request merged**